src/components/userInput/UserResponse.tsx

```diff
@@ -46,41 +46,111 @@ export const UserTextInput = styled('input')(
   }),
 )
 
-const ActualSelectInput = styled('select')({
+const ActualSelectInput = styled('select')(({ hasError }: InputProps) => ({
   appearance: 'none',
   background: 'transparent',
   fontSize: 'inherit',
   fontWeight: 'inherit',
   paddingRight: '1.5em',
   border: 'none',
   cursor: 'pointer',
+  color: hasError ? colors.PINK : undefined,
+}))
+
+const UserSelectInputWrapper = styled(UserTextInput)(
+  ({ hasError }: WithMaxWidth & InputProps) => ({
+    appearance: 'none',
+    cursor: 'pointer',
+    position: 'relative',
+    '&::after': {
+      borderStyle: 'solid',
+      borderWidth: '.1em .1em 0 0',
+      borderColor: hasError ? colors.PINK : colors.OFF_BLACK,
+      content: '" "',
+      position: 'absolute',
+      zIndex: -1,
+      height: '.45em',
+      width: '.45em',
+      right: '.3em',
+      top: '15%',
+      transform: 'rotate(135deg) translateY(-15%)',
+    },
+  }),
+).withComponent('label')
+
+export const UserSelectInput = (
+  props: React.SelectHTMLAttributes<HTMLSelectElement> & InputProps,
+) => (
+  <UserSelectInputWrapper hasError={props.hasError}>
+    <ActualSelectInput {...props} />
+  </UserSelectInputWrapper>
+)
+
+const CheckboxWrapper = styled('span')(
+  ({ disabled }: { disabled?: boolean }) => ({
+    opacity: disabled ? 0.3 : 1,
+    display: 'inline-flex',
+    flexDirection: 'row',
+    alignItems: 'center',
+  }),
+)
+const CheckboxInputContainer = styled('span')({
+  display: 'inline-flex',
+  alignItems: 'center',
+  justifyContent: 'center',
+  cursor: 'pointer',
+  position: 'relative',
 })
+const Tick = styled('svg')({
+  opacity: 0,
+  transition: 'opacity 150ms',
+  position: 'absolute',
+  top: '50%',
+  left: '50%',
+  width: '.6em',
+  transform: 'translateX(-50%) translateY(-50%)',
+  fill: colors.WHITE,
 
-const UserSelectInputWrapper = styled(UserTextInput)((_: WithMaxWidth) => ({
+  'input:checked + &': {
```

Comment:
Would be nicer to use react than CSS magic to do this, change it if you want :)

Reply:
Why? :) it's a pure style thing so imo it should be css